src/amount.h

```diff
@@ -25,4 +25,12 @@ static const CAmount COIN = 100000000;
 static const CAmount MAX_MONEY = 21000000 * COIN;
 inline bool MoneyRange(const CAmount& nValue) { return (nValue >= 0 && nValue <= MAX_MONEY); }
 
+/** Check if an amount is a valid fee delta.
+ *
+ * Like MoneyRange(...), but allows for negative money amounts.
+ */
+inline bool FeeDeltaRange(const CAmount fee_delta) {
+    return fee_delta >= -MAX_MONEY && fee_delta <= MAX_MONEY;
```

Comment:
Fee deltas outside `MAX_MONEY` are valid and have at least a conceptual use case (consider that fee rate is divided by transaction size: a larger transaction would need `>MAX_MONEY` feerate to go before a `MAX_MONEY` smaller tx).

Reply:
@luke-jr What is the valid fee delta range in your opinion?

Reply:
I'm not sure this is actionable unless we can establish what the valid range should be.

Reply:
This is not a feerate, but an absolute amount in satoshis